objectRecognitionSource/src/test/java/io/vantiq/extsrc/objectRecognition/neuralNet/TestNoProcessor.java

```diff
@@ -54,7 +55,7 @@ public class TestNoProcessor extends NeuralNetTestBase {
 
     // A single processor is used for the entire class because it is very expensive to do initial setup
     @BeforeClass
-    public static void classSetup() {
+    public static void classSetup() throws Exception {
```

Comment:
Not a big deal since you've already made the changes, but for future reference you can use Lombok's `@SneakyThrows` to avoid this when it is just extra bookkeeping.